### PR TITLE
New dataset: Sudan Blue Nile CEO 2020

### PR DESCRIPTION
**Pull request by MsPixels**
Add Sudan Blue Nile CEO 2020 data

### CODE REVIEW

File: datasets.py [hunk 1]
```diff
@@ -122,6 +122,36 @@ def load_labels(self) -> pd.DataFrame:
...
+            {
+                CLASS_PROB: "mean",
+                "num_labelers": "sum",
+                "plotid": join_unique,
+                "sampleid": join_unique,
+                "email": join_unique,
+            }
+        )
+        df[START], df[END] = date(2020, 1, 1), date(2021, 12, 31)
+        df[SUBSET] = train_val_test_split(df.index, 0.5, 0.5)
```
Comment (ivanzvonkov) [inline datasets.py:151]: There's 1500 points here, I think we can put less into the validation and test set, maybe `0.35, 0.35`

Comment (MsPixels) [reply datasets.py:151]: Alright Ivan

**Comment by ivanzvonkov** [comment]
Reset the `dataset.dvc` file to ensure the new split is created. See updated instructions here: https://github.com/nasaharvest/crop-mask/wiki/3.-Adding-labeled-data

**Comment by ivanzvonkov** [comment]
Will approve one all processing is done

**Review by ivanzvonkov — APPROVED** [review]
Good to merge if all tests pass!

**Review by ivanzvonkov — APPROVED** [review]

**Pull request merged**